test/pstl_offload/headers/headers.dpl/oneapi_dpl_dynamic_selection.pass.cpp

```diff
@@ -0,0 +1,17 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) 2023 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <oneapi/dpl/dynamic_selection>
+#include "support/utils.h"
+
+int main() {
+    using namespace oneapi::dpl::experimental;
+    [[maybe_unused]] std::size_t r = sizeof(fixed_resource_policy<sycl_policy>);
```

Comment:
I don't remember `sycl_policy` API in dynamic selection. Are you sure this test passes?

Reply:
Fixed, should be `sycl_backend`